src/resolvers.ts

```diff
@@ -55,11 +55,16 @@ export const resolvers = {
         throw new UnauthorizedError('Credenciais inválidas');
       }
 
+      const rememberMe = args.input.rememberMe;
       const token = Jwt.sign({ userId: user.id }, process.env.JWT_TOKEN as string, { expiresIn: '1h' });
+      const refreshToken = Jwt.sign({ userId: user.id }, process.env.JWT_TOKEN as string, { expiresIn: '7d' });
+      if (rememberMe) {
+        return { user, token: refreshToken, refreshToken, rememberMe };
```

Comment:
não precisa retornar um campo `refreshToken`, esse campo não faz parte do seu schema
```suggestion
        return { user, token: refreshToken, rememberMe };
```